### DISCUSSION TITLE
Mod List Widget

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#658] [Suggestions]
Hey Max,

Would it be possible to have a widget/content block for the website that shows the current active mods on a server, including links to click on to take you to the mod page. This will help for ASA servers where we are unable to create mod collections on steam.

Thank you,
Gem

**Comment by Decollates** [comment]
I would absolutely love a feature like this.

**Reply by babsgosgens** [reply]
Ditto, +1 from me as well.

**Comment by mvaessen** [comment]
Thanks for the suggestion, this is now available!
